helpers/traces/devtools-timeline-model.js

```diff
@@ -0,0 +1,164 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+/* global WebInspector, TimelineModelTreeView */
+
+console.log('Booting dtm');
+
+// establish our sandboxed globals
+if (typeof global.window === 'undefined') {
+  global.window = global.self = global;
+}
+
+// global.WebInspector = global.WebInspector || {};
+global.Runtime = global.Runtime || {};
+// global.TimelineModelTreeView = global.TimelineModelTreeView || {};
+global.TreeElement = global.TreeElement || {};
+global.WorkerRuntime = global.WorkerRuntime || {};
+// global.Protocol = global.Protocol || {};
```

Comment:
No I just uncommented the ones it complained about directly. I went conservative because of the clashes with Network Manager.